productivity-suite/piercing-library/src/piercing-gateway.ts

```diff
@@ -100,14 +99,6 @@ export class PiercingGateway<Env> {
     this.fragmentConfigs.set(fragmentConfig.fragmentId, fragmentConfig);
   }
 
-  /*
-		We're assigning fetch to the object itself so that user can do:
-			`export default gateway;`
-		this shouldn't be necessary and we should be able to have fetch
-		as a normal method in the class, but that doesn't seem to
-		currently being recognized correctly:
-			https://chat.google.com/room/AAAADFUqSgo/njWMATzSHG4
```

Comment:
this refers to a private gchat conversation so I think we can/should just exclude it (it is also not an issue related to the piercing strategy)

Reply:
Actually, I think the main comment is a valid thing to keep here. Perhaps just remove the link to the gchat?

Reply:
I've put the comment back 🙂